end-to-end/package-jazzer-js.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+cd ..
+npm install
+npm run build
+npm run build --workspace='@jazzer.js/fuzzer'
+
+sed_version_and_mv() {
+    while read data; do
+        local no_version=$(echo $data | sed -r -f end-to-end/remove-version.sed)
+        echo "mv $data end-to-end/$no_version"
+        mv $data end-to-end/$no_version
+    done
+}
+
+npm pack --workspaces | sed_version_and_mv 
+
+cd examples
+
+npm install
```

Comment:
I think `npm install --save-dev *.tgz` installs everything at once.

Reply:
Ah, this showed me a few mistakes. This command probably doesn't belong in a script called `package-jazzer-js` anyway. I also realized that `npm install --save-dev *.tgz` is actually required because `npm install` will ignore new tarballs if you've already installed them even if you've deleted `node_modules`.